### ISSUE TITLE
Problem updating by winget on Windows 

### ISSUE THREAD

**Issue by mko61** [timschneeb/GalaxyBudsClient#431]
Launching an update via winget, it constantly find the installed version is different from the git hub version:

Name                                                                             Id                                                                   Available Version
-------------------------------------------------------------------------------------------------------------------------------
Galaxy Buds Manager (Unofficial) 4.5.4-1dfedf (64bit) ThePBone.GalaxyBudsClient 4.5.4-1dfedf       4.5.4

The package from the Windows installer has a "-1dfedf" at the end of the id that triggers the difference. 
The installer available on github and the one downloaded by winget are the same.

Obviously, versions should be the same.

**Comment by Joedmin** [comment]
I can see that the package is missing the commit sem ver label in the [winget pacages](https://github.com/microsoft/winget-pkgs/tree/master/manifests/t/ThePBone/GalaxyBudsClient). I will open a PR there to fix it

**Comment by cebu23** [comment]
This is still a problem, and fairly easy to recreate (for me, your mileage may differ).  

Start with a system that does not have GalaxyBudsClient installed.  Then install it as shown below - following the successful install, a list command will show the version mismatch, and update will re-download and re-install even though you just installed the latest copy a moment ago.
```
PS C:\Users\chris> winget install --id=ThePBone.GalaxyBudsClient -e
Found Galaxy Buds Manager (Unofficial) [ThePBone.GalaxyBudsClient] Version 4.5.4
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/ThePBone/GalaxyBudsClient/releases/download/4.5.4/GalaxyBudsClient_Windows_64bit_Setup.exe
  ██████████████████████████████  47.0 MB / 47.0 MB
Successfully verified installer hash
Starting package install...
Successfully installed
PS C:\Users\chris> winget list --id=ThePBone.GalaxyBudsClient -e
Name                                                  Id                        Version      Available Source
-------------------------------------------------------------------------------------------------------------
Galaxy Buds Manager (Unofficial) 4.5.4-1dfedf (64bit) ThePBone.GalaxyBudsClient 4.5.4-1dfedf 4.5.4     winget
PS C:\Users\chris> winget update --id=ThePBone.GalaxyBudsClient -e
Found Galaxy Buds Manager (Unofficial) [ThePBone.GalaxyBudsClient] Version 4.5.4
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/ThePBone/GalaxyBudsClient/releases/download/4.5.4/GalaxyBudsClient_Windows_64bit_Setup.exe
  ██████████████████████████████  47.0 MB / 47.0 MB
Successfully verified installer hash
Starting package install...
Successfully installed
PS C:\Users\chris>
```

A tangential error to point out - the latest version available via winget is 4.5.4 (a.k.a. 4.5.4-1dfedf), it hasn't been updated to 4.6.0 yet.

**Comment by prakharb5** [comment]
The installer is not able to save the version details in the system at all now :/
Due to this, winget does not know which version is installed. Please look into this 

Even though I just installed the latest version (5.0.1), it still does not know that.
![image](https://github.com/timschneeb/GalaxyBudsClient/assets/51435948/23f177bd-4250-48b8-8f73-de77e62f3075)

**Comment by timschneeb** [comment]
This will be fixed in the next version. A while back, my CI build script stopped setting the correct version information.
(Fixed in commit 85b88b58a8e203d37ce5ec6f626ed1b6b06d63e6)

**Comment by timschneeb** [comment]
Fixed, see https://github.com/timschneeb/GalaxyBudsClient/issues/522#issuecomment-2232770855
